src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -20,9 +21,11 @@ public sealed partial class QuicConnection : System.IDisposable
         public IPEndPoint RemoteEndPoint => throw null;
         public QuicStream OpenUnidirectionalStream() => throw null;
         public QuicStream OpenBidirectionalStream() => throw null;
+        public long GetMaximumBidirectionalStreamCount() => throw null;
+        public long GetMaximumUnidirectionalStreamCount() => throw null;
         public System.Threading.Tasks.ValueTask<QuicStream> AcceptStreamAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
         public System.Net.Security.SslApplicationProtocol NegotiatedApplicationProtocol => throw null;
-        public ValueTask CloseAsync(System.Threading.CancellationToken cancellationToken = default) => throw null;
+        public ValueTask CloseAsync(long shutdownCode, System.Threading.CancellationToken cancellationToken = default) => throw null;
```

Comment:
`shutdownCode` might give users the wrong idea (thinking relation to `Socket.Shutdown`) -- QUIC spec calls it `errorCode` even when successful (`NO_ERROR`) so maybe that.